settings.yaml

```diff
@@ -0,0 +1,47 @@
+# Vagrant and Kubernetes default configuration
+# 
+# The settings.yaml is the default configuration file loaded by Mage and Vagrant.
+#
+# Defaults can be overriden using modified copy of this file passed via either
+# 1. Environment variable SWDT_SETTINGS_FILE=my.yaml
+# 2. Naming user-specific file as settings.local.yaml
+
+## Vagrant machines
+# Linux control plane
+vagrant_linux_box: "mloskot/sig-windows-dev-tools-ubuntu-2204"
```

Comment:
Do we really want this to be configurable? We really don't want users trying to use random Vagrant machines and then discovering things don't work as needed.